i/logo.svg

```diff
@@ -0,0 +1,51 @@
+<svg
+  class="crm-logo"
+  aria-label="[title + description]"
+  role="img"
+  version="1.1"
+  viewBox="0 0 229.76 59.783"
+  xml:space="preserve"
+  xmlns="http://www.w3.org/2000/svg"
+  >
+  <title>{ts}CiviCRM Logo{/ts}</title>
+  <description>{ts}The word CiviCRM with an intersecting blue and green triangle{/ts}</description>
+  <style type="text/css">
+    .crm-logo-text {
+      fill: var(--crm-c-logo-text, #000);
```

Comment:
```suggestion
      fill: var(--crm-c-logo-text, #086287);
```

actioning your suggestion @artfulrobot ?
 
hurrah for line-by-line editable images ;)